test/functional/Java8andUp/src_80/org/openj9/test/java/lang/Test_Thread_Java8to10.java

```diff
@@ -0,0 +1,278 @@
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+import org.testng.Assert;
+import org.testng.AssertJUnit;
+import java.lang.ref.WeakReference;
+
+@Test(groups = { "level.sanity" })
+public class Test_Thread_Java8to10 {
+
+	static class SimpleThread implements Runnable {
+		int delay;
+
+		public void run() {
+			try {
+				synchronized (this) {
+					this.notify();
+					this.wait(delay);
+				}
+			} catch (InterruptedException e) {
+				return;
+			}
+
+		}
+
+		public SimpleThread(int d) {
+			if (d >= 0)
+				delay = d;
+		}
+	}
+
+	static class YieldThread implements Runnable {
+		volatile int delay;
+
+		public void run() {
+			int x = 0;
+			while (true) {
+				++x;
+			}
+		}
+
+		public YieldThread(int d) {
+			if (d >= 0)
+				delay = d;
+		}
+	}
+
+	static class ResSupThread implements Runnable {
+		Thread parent;
+		volatile int checkVal = -1;
+
+		public void run() {
+			try {
+				synchronized (this) {
+					this.notify();
+				}
+				while (true) {
+					checkVal++;
+					zz();
+					Thread.sleep(100);
+				}
+			} catch (InterruptedException e) {
+				return;
+			} catch (BogusException e) {
+				try {
+					// Give parent a chance to sleep
+					Thread.sleep(500);
+				} catch (InterruptedException x) {
+				}
+				parent.interrupt();
+				while (!Thread.currentThread().isInterrupted()) {
+					// Don't hog the CPU
+					try {
+						Thread.sleep(50);
+					} catch (InterruptedException x) {
+						// This is what we've been waiting for...don't throw it
+						// away!
+						break;
+					}
+				}
+			}
+		}
+
+		public void zz() throws BogusException {
+		}
+
+		public ResSupThread(Thread t) {
+			parent = t;
+		}
+
+		public synchronized int getCheckVal() {
+			return checkVal;
+		}
+	}
+
+	static class BogusException extends Throwable {
+		public BogusException(String s) {
+			super(s);
+		}
+	}
+
+	Thread st, ct, spinner;
+
+	static boolean calledMySecurityManager = false;
+
+	// used in test_defaultMemoryArea to pass status from thread started
+	// back to thread running test case
+	boolean pass = true;
+
+	/**
+	 * @tests java.lang.Thread#destroy()
+	 */
+	@Test
+	public void test_destroy() {
+		try {
+			new Thread().destroy();
+			Assert.fail("NoSuchMethodError expected");
+		} catch (NoSuchMethodError e) {
+			// expected
+		}
+	}
+
+	/**
+	 * @tests java.lang.Thread#stop(java.lang.Throwable)
+	 */
+	@Test
+	public void test_stop3() {
```

Comment:
I don't think we need this test, test_stop6() is sufficient. test_stop3() seems like a modified version of the test that used to test SecurityManager behavior but now doesn't really test anything.